.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Please start by choosing an appropriate base branch for your pull request.
+
+Making a quick fix for a problem observed on production service? Fixing a minor
+bug, improving existing documentation or record metadata? Make your pull request
+against the ``production`` branch.
+
+Introducing a new feature? Make your pull request against the ``master`` branch.
+
+Please reference all issues that this PR addresses or closes. Do not forget to
+include ``(addresses #123)`` or ``(closes #123)`` in your commit log message.
+
+Please run the tests locally in order to speed up developments and save Travis
+CI cycles.  Look for ``./run-tests.sh``.
```

Comment:
Again, not all of us will run these tests, so how do we balance this?

Reply:
It wouldn't apply in that case... but somebody (i.e. the integrator) will have to run the tests in those cases.